client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -1,26 +1,45 @@
 import { sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
+import PropTypes from 'prop-types';
 import React from 'react';
 import { useInterval } from 'calypso/lib/interval/use-interval';
 import './style.scss';
 
 // Total time to perform "loading"
 const DURATION_IN_MS = 6000;
 
+const useSteps = ( flowName, { hasPaidDomain, hasSelectedDesign } ) => {
```

Comment:
You could also perhaps just forward all of the props as a single `props` object to the hook, since it's the only thing that's actually consuming the component's props 🤷‍♂️ 

```suggestion
const useSteps = ( { flowName, hasPaidDomain, hasSelectedDesign } ) => {
```